analytics/client.go

```diff
@@ -0,0 +1,143 @@
+package analytics
+
+import (
+	"github.com/dukex/mixpanel"
+	"github.com/golang/glog"
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+	"github.com/segmentio/analytics-go/v3"
+)
+
+type Client interface {
+	// Sends the given tracking event to Segment and Mixpanel (if enabled).
+	Track(distinctID string, event *Event) error
+}
+
+type clientImpl struct {
+	config        Config
+	segmentClient analytics.Client
```

Comment:
I'd import the Segment package as `segment` to avoid confusing `analytics.Client` with the `Client` interface defined above.

Reply:
Updated in [f98a5fb](https://github.com/akitasoftware/akita-libs/pull/152/commits/f98a5fb1c9d9ad9fc3ebe457f7c38611db39b2b8)